ui.content/src/main/content/jcr_root/content/experience-fragments/venia/us/en/site/products/productlist-content/master/.content.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2021-10-11T14:30:58.039+02:00"
+        cq:lastModifiedBy="admin"
+        cq:categories="MTQ="
```

Comment:
The category uid is not available in the adobe commerce environment we use for our automated testing. It should be `Mzc=`